var/spack/repos/builtin/packages/py-efel/package.py

```diff
@@ -26,19 +26,20 @@
 
 
 class PyEfel(PythonPackage):
-    """The Electrophys Feature Extract Library (eFEL) allows 
-    neuroscientists to automatically extract features from time series data 
+    """The Electrophys Feature Extract Library (eFEL) allows
+    neuroscientists to automatically extract features from time series data
     recorded from neurons (both in vitro and in silico).
-    Examples are the action potential width and amplitude in 
+    Examples are the action potential width and amplitude in
     voltage traces recorded during whole-cell patch clamp experiments.
-    The user of the library provides a set of traces and selects the 
-    features to be calculated. The library will then extract the requested 
+    The user of the library provides a set of traces and selects the
+    features to be calculated. The library will then extract the requested
     features and return the values to the user."""
     homepage = "https://github.com/BlueBrain/eFEL"
-    url = "https://pypi.io/packages/source/e/efel/efel-3.0.22.tar.gz"
+    url = "https://pypi.io/packages/source/e/efel/efel-3.0.70.tar.gz"
 
+    version('3.0.70')
```

Comment:
needs a sha still. `spack fetch …` will complain with the right sha (if you start by copying the old one)